README.md

```diff
@@ -357,6 +386,58 @@ _Note: this option is **not valid** if you are planning to take Azure snapshots
 
 Install Velero, including all prerequisites, into the cluster and start the deployment. This will create a namespace called `velero`, and place a deployment named `velero` in it.
 
+### For Latest Azure Plugin >1.7.0
```

Comment:
How about organizing the content as follows?
```
If using service principal or AAD Pod Identity:
...
  For Latest Azure Plugin >=1.8.0
    ...
  For Azure Pluing <1.8.0
   ...

If using storage account access key and no Azure snapshots 
...
```

Reply:
Ack will do